cmd/fleetctl/get_test.go

```diff
@@ -826,13 +826,29 @@ spec:
   platform: darwin
   targets:
     labels: null
-`
-	expectedJson := `{"kind":"pack","apiVersion":"v1","spec":{"id":7,"name":"pack1","description":"some desc","platform":"darwin","disabled":false,"targets":{"labels":null}}}
+    teams: null
+`
+	expectedJson := `
+{
+  "kind": "pack",
+  "apiVersion": "v1",
+  "spec": {
+    "id": 7,
+    "name": "pack1",
+    "description": "some desc",
+    "platform": "darwin",
+    "disabled": false,
+    "targets": {
+      "labels": null,
+      "teams": null
+    }
+  }
+}
 `
 
 	assert.Equal(t, expected, runAppForTest(t, []string{"get", "packs"}))
-	assert.Equal(t, expectedYaml, runAppForTest(t, []string{"get", "packs", "--yaml"}))
-	assert.Equal(t, expectedJson, runAppForTest(t, []string{"get", "packs", "--json"}))
+	assert.YAMLEq(t, expectedYaml, runAppForTest(t, []string{"get", "packs", "--yaml"}))
+	assert.JSONEq(t, expectedJson, runAppForTest(t, []string{"get", "packs", "--json"}))
```

Comment:
Oh, so the problem with these iirc is that when it fails, it's pretty unreadable because of how it compares them. In contrast, the other way it's a string diff, so it's super clear where it's problematic.

Granted, not the json one... the yaml one.

Have you checked that?